libraries/api-cat-common/src/main/java/no/fdk/acat/common/model/ApiEditableProperties.java

```diff
@@ -39,4 +40,10 @@
     private String availability;
 
     private String statusCode;
+
+    private Date deprecatedExpirationDate;
```

Comment:
Mye bedre navn er noe slikt som 
ExpirationDateForDeprecation
MessageForDeprecation
ReplaceWithUrlWhenDeprecated.
For å indikere at dette er data om APIen det er snakk om, IKKE at disse feltene er deprekert og skal bort senere.

Reply:
jeg foreslår prefix ikke suffix, så at når vi har bedre PATCH protokol, er det lettere å forstå hva må ekstraheres..

hva tenker du om:
```
deprecationInfoExpirationDate
deprecationInfoMessage
deprecationInfoReplacementUrl /  deprecationInfoReplacedWithUrl
```

Reply:
ReplacedWithUrl er best tenker jeg.